From 4e5f8cbd74a22c4018526e30b16bb4824e5c06a1 Mon Sep 17 00:00:00 2001
From: Viktoria Maximova <viktoria.maksimova@intel.com>
Date: Fri, 28 Apr 2023 07:29:16 -0700
Subject: [PATCH 117/172] [Backport to 14][NFC][DebugInfo] Rewrite
 NonSemantic/DebugInfoProducer test

There is a bug in -r -spirv-text mode in translation of Strings with spaces.
Temporary avoid running such pipeline until proper fix is delivered.
---
 test/DebugInfo/NonSemantic/DebugInfoProducer.ll | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/test/DebugInfo/NonSemantic/DebugInfoProducer.ll b/test/DebugInfo/NonSemantic/DebugInfoProducer.ll
index a3a8e3c5..27ef3b2c 100644
--- a/test/DebugInfo/NonSemantic/DebugInfoProducer.ll
+++ b/test/DebugInfo/NonSemantic/DebugInfoProducer.ll
@@ -20,8 +20,9 @@
 ; Check that we don't produce "producer" info for NonSemantic.Shader.DebugInfo.100 as it's not specification conformant
 ; RUN: llvm-spirv %t.bc --spirv-debug-info-version=nonsemantic-shader-100 -spirv-text -o %t.spt
 ; RUN: FileCheck %s --input-file %t.spt --check-prefix CHECK-SPIRV-100
-; RUN: llvm-spirv -spirv-text -r --opaque-pointers %t.spt -o %t.rev.bc
-; RUN: llvm-dis -opaque-pointers %t.rev.bc -o - | FileCheck %s --check-prefix CHECK-LLVM-100
+; RUN: llvm-spirv %t.bc --spirv-debug-info-version=nonsemantic-shader-100 -o %t.spv
+; RUN: llvm-spirv -r %t.spv -o %t.rev.bc
+; RUN: llvm-dis %t.rev.bc -o - | FileCheck %s --check-prefix CHECK-LLVM-100
 
 ; ModuleID = 's.bc'
 source_filename = "s.cpp"
-- 
2.20.1

